examples/graphbolt/node_classification.py

```diff
@@ -309,6 +310,9 @@ def train(args, graph, features, train_set, valid_set, num_classes, model):
             # in the last layer's computation graph.
             y = data.labels
 
+            if y.dtype != "int64":
+                y = y.long()
```

Comment:
```suggestion
            y = data.labels.long()
```

Reply:
This already only converts if it is not int64.

Reply:
Is this conversion needed by the loss function?

Reply:
Yes. Originally, the y have dtype of `float32` and the `crossEntropy` func will give an Exception on `expected scalar type long but found float`. Is there other ways to get around this instead of type casting? Or it is ok to do the cast?

Reply:
Why not cast the labels before saving them to the file?

Reply:
Yes. I can also do that in the download.py